src/api/config.rs

```diff
@@ -297,13 +304,13 @@ impl SpeedSettings {
   /// - 9: min block size 64x64, reduced TX set, TX domain distortion, fast deblock.
   /// - 8: min block size 8x8, reduced TX set, TX domain distortion, fast deblock.
   /// - 7: min block size 8x8, reduced TX set, TX domain distortion.
-  /// - 6: min block size 8x8, reduced TX set, TX domain distortion.
-  /// - 5 (default): min block size 8x8, reduced TX set, TX domain distortion, complex pred modes for keyframes.
-  /// - 4: min block size 8x8, TX domain distortion, complex pred modes for keyframes.
-  /// - 3: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision.
-  /// - 2: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision, include near MVs, quantizer RDO.
-  /// - 1: min block size 8x8, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO.
-  /// - 0 (slowest): min block size 4x4, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO, bottom-up encoding.
+  /// - 6 (default): min block size 8x8, reduced TX set, TX domain distortion, complex pred modes for keyframes.
+  /// - 5: min block size 8x8, TX domain distortion, complex pred modes for keyframes.
+  /// - 4: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision.
+  /// - 3: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision, include near MVs, quantizer RDO.
+  /// - 2: min block size 8x8, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO.
+  /// - 1: min block size 4x4, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO, bottom-up encoding.
+  /// - 0 (slowest): min block size 4x4, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO, bottom-up encoding with non-square partitions not just right and bottom tile borders but everywhere
```

Comment:
It's a bit verbose to include "not just right and bottom tile borders but everywhere" and I think just saying "with non-square partitions" communicates the intent

Reply:
Sure- np. Done.